docs/book/src/reference/metrics.md

```diff
@@ -21,6 +21,23 @@ You can run the following kubectl command to create it. If using kubebuilder
 kubectl create clusterrolebinding metrics --clusterrole=<project-prefix>-metrics-reader --serviceaccount=<namespace>:<service-account-name>
 ```
 
```

Comment:
This is some slight modification to Line 17
You can either run the following command, or apply the sample yaml file to create `clusterRoleBinding`. If using kubebuilder
`<project-prefix>` is the `namePrefix` field in `config/default/kustomization.yaml`.